extensions/arc/src/localizedConstants.ts

```diff
@@ -161,6 +162,7 @@ export const details = localize('arc.details', "Details");
 export const lastUpdated = localize('arc.lastUpdated', "Last updated");
 export const noExternalEndpoint = localize('arc.noExternalEndpoint', "No External Endpoint has been configured so this information isn't available.");
 export const podsReady = localize('arc.podsReady', "pods ready");
+export const connectToPostgresDescription = localize('arc.connectToPostgresDescription', "A connection to server is required to show and set database engine settings. Make sure to have installed PostgreSQL extension for Azure Data Studio.");
```

Comment:
We should be checking for this and if it isn't installed provide a link in the toast notification for them to install it. You can :


1. Create a new command to install the extension (let's say you call it arc.installPostgreSQLExtension) that calls the workbench.extensions.installExtension command with the extension ID
1. Then have the notification be something like this : 

`The PostgreSQL extension is required to connect to this server. [Click here](command:arc.installPostgreSQLExtension) to install it` and then try again.`

Reply:
This shows above connect button so that user is aware of why connection is needed and what is required for it. A check for installed extension happens once they attempt to connect.